ddp-workspace/projects/ddp-sdk/src/lib/components/subjectPanel.component.ts

```diff
@@ -0,0 +1,39 @@
+import { Component, Inject } from '@angular/core';
+import { SessionMementoService } from '../services/sessionMemento.service';
+import { ConfigurationService } from '../services/configuration.service';
+
+@Component({
+    selector: 'ddp-subject-panel',
```

Comment:
We use this panel on 3 pages - in the Toolkit, SDK, and Testboston app, I decided to make it as SDK's component.